src/python/pants/init/specs_calculator.py

```diff
@@ -102,12 +120,18 @@ def create(
         if not changed_options.provided:
             return specs
 
-        scm = get_scm()
-        if not scm:
-            raise InvalidSpecConstraint(
-                "The `--changed-*` options are not available without a recognized SCM (usually "
-                "Git)."
-            )
+        def scm(entity):
+            scm = get_scm()
+            if not scm:
+                raise InvalidSpecConstraint(
+                    # TODO: centralize the error messaging for when an SCM is required, and describe what SCMs
+                    # are supported!
+                    "{} are not available without a recognized SCM (usually git).".format(entity)
```

Comment:
Maybe related: the sole SCM that Pants supports right now is Git. I think we'll have better error messages if we reflect that reality and solely mention Git. Users don't need to see the word SCM.

We can keep the actual class SCM of course. And if/when we add a second SCM, we can update these error messages to reflect that new reality. I imagine we will then search for all uses of `get_scm()` + "git", so we'll find this.

Reply:
I don't agree with "users don't need to see the word SCM" -- if a user tries to use pants on their mercurial repo and it fails, their next response is not likely to be to open an issue against the pants repo and expect help if we assume git.

Reply:
I have modified the error message to specify that we currently only support git.